recipe/build.sh

```diff
@@ -15,9 +15,8 @@ fi
 # Install Cycamore
 #export VERBOSE=1
 ${PYTHON} install.py --prefix="${PREFIX}" \
-  --build_type="Release" \
-  --dont-allow-milps \
-  --deps-root="${PREFIX}" \
-  -DCMAKE_OSX_DEPLOYMENT_TARGET="${MACOSX_VERSION_MIN}" \
+  -DCMAKE_C_COMPILER=${GCC} \
+  -DCMAKE_CXX_COMPILER=${CXX} \
+  --allow-milps \
   ${skiprpath} \
-  --clean -j "${CPU_COUNT}"
```

Comment:
Does `--clean` make a difference in this context? If so, then maybe best to keep it?

Reply:
We can leave it in for verbosity if you'd like, but it is building in a fresh environment so the flag should not be necessary.  It was removed in the Cyclus-feedstock as well